source/codegen/metadata/nirfmxnr/attributes.py

```diff
@@ -225,6 +225,11 @@
         'name': 'MODACC_COMPOSITE_RESULTS_INCLUDE_PTRS',
         'type': 'int32'
     },
+    9437244: {
```

Comment:
Are these config changes hand-edits? @reckenro do we still need to keep these in sync with scrapigen? Or is there another system for it?

Reply:
Yes, the files under source/codegen/metadata/nirfmxnr were edited by hand.

Reply:
Yes, we're still sourcing RFmx from [scrapigen](https://github.com/ni/grpc-device-scrapigen). KC did the most recent RFmx update there to update RFmx grpc-device metadata to 22.5 so a similar one should be done for NR to produce the 23.0 metadata. Eventually these updates should be done by the driver teams but I think the ownership of that is still in the transition phase so for now it's been on an as-needed basis.

I made #839  to scatter READMEs in the codegen/metadata/RFmx folders to note that they're sourced from scrapigen because right now there's no indication of that.

Reply:
I updated the metadata with the ones generated by scrapigen. The only difference was two attributes I had incorrectly left out. See [scrapigen PR]([url](https://github.com/ni/grpc-device-scrapigen/pull/102)).